src/Model/Widget/SpanParser.php

```diff
@@ -10,6 +10,12 @@
 use PhpTui\Tui\Model\Modifier;
 use PhpTui\Tui\Model\Style;
 
+/**
+ * This class is a subset of the Symfony Console component's OutputFormatter class.
+ * It parses strings containing tags into a list of spans.
```

Comment:
Technically it's not a subset of the class but a subset of the ... markup?

Reply:
Yeah, you're right. 😅